nym-vpn-core/crates/nym-vpnd/Cargo.toml

```diff
@@ -64,7 +64,7 @@ nym-vpnd-types = { path = "../nym-vpnd-types" }
 [target.'cfg(windows)'.dependencies]
 windows-service = "0.7.0"
 eventlog = "0.3.0"
-winapi = { version = "0.3", features = ["winnt", "excpt"] }
+winapi = { version = "0.3", features = ["winnt", "excpt", "winerror"] }
```

Comment:
FYI Not a problem with this PR but to raise awareness. We've been putting effort at consolidating dependencies in the workspace Cargo.toml so it's easier to keep track of them. IMO features can be specified by each indiviudal crate, that's ok.